browser/src/control/Control.Tooltip.js

```diff
@@ -0,0 +1,122 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * L.Control.Tooltip - tooltip manager
+ */
+
+/* global */
+
+L.Control.Tooltip =  L.Control.extend({
```

Comment:
We currently try to get rid of Leaflet. Also we want new code in TypeScript.
Could you convert that into regular `class` (not inheriting from L.Control), and make this file `.ts` ?

Reply:
Sorry, I am not fan on proprietary library, but I fan standard an free software. I can change it
to class standard javascript

Reply:
Yes, I mean: please use `class` from JavaScript - ECMAScript 6/2015 standard :)